tests/unit/components/buttons/HdLoaderButton.spec.js

```diff
@@ -105,4 +105,23 @@ describe('HdLoaderButton', () => {
 
     expect(clicked).toHaveBeenCalled();
   });
+
+  it('should toggle disabled class', async () => {
+    // Enabled
+    const disabledClass = 'btn--primary-disabled';
+    const wrapper = wrapperBuilder({
+      props: {
+        disabled: false,
+      },
+    });
+    const button = wrapper.get(BUTTON_SELECTOR);
+    expect(button.attributes().class).not.toContain(disabledClass);
```

Comment:
I think you could have used https://vue-test-utils.vuejs.org/api/wrapper/#classes

Reply:
Yeah makes sense, done 👍